### ISSUE TITLE
How to split MOT17 into training set and validation set?

### ISSUE THREAD

**Issue by ljd16** [ifzhang/FairMOT#314]
Good work!
When reading your paper and code, I found that you did not explain in detail how to divide MOT17 into training set and validation set, so can you provide more details?
mot17.val and mot17.half divide the same sequence into two parts, which is incorrect.

**Comment by Cardinal2376** [comment]
Also curious， did you find out the detailed method?

**Comment by Cardinal2376** [comment]
> Good work!
> When reading your paper and code, I found that you did not explain in detail how to divide MOT17 into training set and validation set, so can you provide more details?
> mot17.val and mot17.half divide the same sequence into two parts, which is incorrect.

Also curious， did you find out the detailed method?

**Comment by ljd16** [comment]
> > Good work!
> > When reading your paper and code, I found that you did not explain in detail how to divide MOT17 into training set and validation set, so can you provide more details?
> > mot17.val and mot17.half divide the same sequence into two parts, which is incorrect.
> 
> Also curious， did you find out the detailed method?

I think that mot17.train is the real trainval dataset, mot17.val is the validation dataset, and mot17.half is the train dataset.
